schemas/extractor.yml

```
@@ -65,9 +109,49 @@ classes:
                     extractor.
             usage:
                 multivalued: true
+                inlined_as_list: true
+                range: Usage
                 description: >-
-                    A command-line invocation of the parser, to be available after
-                    installation instructions have been followed.
+                    A machine-actionable description of the usage of the Extractor.
+                    The described usage pattern shall be available after the
+                    'installation' instructions have been followed.
+            installation:
+                multivalued: true
+                inlined_as_list: true
+                range: Installation
+                description: >-
+                    A machine-actionable set of installation instructions to obtain
+                    a working set-up of the Extractor.
             instructions:
                 description: >-
-                    Any usage or installation instructions for this extractor.
+                    Any usage notes or installation instructions for this Extractor.
+                    This field is intended for human use only and is not intended
+                    to be machine-actionable. Please use the 'installation' and
+                    'usage' fields for that purpose.
+
+
+enums:
+    UsageTypes:
+        permissible_values:
+            cli:
+                description: >-
+                    Access the Extractor as a command line executable. The executable
+                    as well as the invocation syntax, parametrised using arguments,
+                    shall be supplied in the 'command' slot.
+            python:
+                description: >-
+                    Access the Extractor as a Python function. The function should
+                    be available when 'import Extractor' is called, and should be
```

```suggestion
                    be available when `import Extractor` is called, and should be
```

I still think this is a bit unclear, I guess now this should be `import <setup>`?

---

```suggestion
                    be available when `import <setup>` is called, and should be
```